core/node/commitment_generator/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 use std::time::Duration;
 
 use anyhow::Context;
+use input_generation::InputGenerator;
```

Comment:
Nit: Please use `crate::input_generation` here instead to clearly indicate that we're using local module and not external crate.